app/src/main/kotlin/at/bitfire/davdroid/ui/account/AccountScreenModel.kt

```diff
@@ -183,13 +185,26 @@ class AccountScreenModel @AssistedInject constructor(
     }
 
     fun setCollectionSync(id: Long, sync: Boolean) {
-        viewModelScope.launch {
+        viewModelScope.launch(Dispatchers.IO) {
```

Comment:
Do we need to specify the dispatcher here? In general, [coroutines should switch context to another dispatcher themselves if required](https://developer.android.com/kotlin/coroutines/coroutines-best-practices#main-safe).

Reply:
Can't access the DB on main thread in `syncAfterDelay` otherwise, but I can use `withContext` in `syncAfterDelay` instead.